manifests/params.pp

```diff
@@ -31,6 +31,12 @@
     $service_enable = true
   }
 
+  if ($::operatingsystem == 'Ubuntu' and $::operatingsystemrelease in ['15.04','15.10','16.04','16.10']) or ($::operatingsystem == 'Debian' and $::operatingsystemrelease == '8') {
```

Comment:
It might be better to simplify this to use the systemd fact:

https://github.com/camptocamp/puppet-systemd/blob/master/lib/facter/systemd.rb

This would mean this patch would also work on RHEL flavoured systems with systemd (Centos 7, Oracle Linux 7, Fedora 14+ etc)